web-app/js/portal/prototypes/NonCollapsingAccordionLayout.js

```diff
@@ -30,7 +29,16 @@ Ext.ux.NonCollapsingAccordionLayout = Ext.extend( Ext.layout.Accordion, {
     beforeExpandPanel : function(panel) {
         var panelToCollapse = this.currentlyExpandedPanel;  // A holder for the previously selected panel
         this.currentlyExpandedPanel = panel;                // Set the new panel as the currently expanded one
-        panelToCollapse.collapse();                         // Collapse the previously selected panel
+
+        if (panelToCollapse.collapsed) {
+            // fixes #1823
+            setTimeout(function() {
+                panelToCollapse.collapse();
+            }, 400 );
+        }
+        else {
+            panelToCollapse.collapse();                     // Collapse the previously selected panel
```

Comment:
No problems, @pmbohm. You're right about this code having changed a lot from the original, now. I might take this opportunity to have it match our code styles, too.

I'll get one of the other devs to finish this PR with me.